### PR TITLE
mpmath dependency: Fixed 'sys' NameError

### PR DESCRIPTION
**Pull request by lokeshh**
Fixed unwanted 'sys' NameError when mpmath dependency is not met 
by placing 'import sys' before mpmath dependency check.

### CODE REVIEW

**Comment by asmeurer** [comment]
+1

File: setup.py [hunk 1]
```diff
@@ -29,6 +29,13 @@
 sympy@googlegroups.com and ask for help.
 """
 
+import sys
+import subprocess
+import os
+import shutil
+import glob
+
+PY3 = sys.version_info[0] > 2
```
Comment (toolforger) [inline setup.py]: This should stay below with the "right Python version" part, below.

**Comment by smichr** [comment]
Thanks! Is there a name that we can add to the AUTHORS for you? I don't believe you are listed there yet.

**Comment by lokeshh** [comment]
Yeah sure, its "Lokesh Sharma" (username: lokeshh).

I believe I hadn't configured the git properly, probably that's why it didn't showed any name up there. That has been sorted out best to my knowledge of git and hopefully it would show my name in future.

Thanks for the wonderful [Development Workflow](https://github.com/sympy/sympy/wiki/Development-workflow) guide you guys have created. It lead me to set up git within a day and help me started in such a short time.

**Comment by smichr** [comment]
I'm glad the workflow helped. I recall how much @certik's suggestions for .config aliases helped me, and watching the debug session also helped to encourage efficient work habits, too.

**Comment by smichr** [comment]
> Yeah sure, its "Lokesh Sharma" (username: lokeshh).

Should I use the `your_email@youremail.com` as the mailing address or was that part of the mistaken setup?

**Comment by certik** [comment]
Thanks for the fix @lokeshh and thanks for merging @smichr. I am glad the videos were useful.

**Comment by lokeshh** [comment]
No, kindly use `lokeshhsharma@gmail.com` as the mailing address. You are right, `your_email@youremail.com` was part of the mistaken setup.

**Pull request merged**